ggu/main.c

```diff
@@ -400,11 +555,11 @@ run_ggu(struct net_config *net_conf,
 		goto out;
 	}
 
-	ret = net_launch_at_stage1(net_conf, 0, 0, 1, 0, ggu_state1, ggu_conf);
+	ret = net_launch_at_stage1(net_conf, 0, 0, 1, 0, ggu_stage1, ggu_conf);
```

Comment:
Previously many of the GT-GK Unit functions were named `ggu_state` instead of `ggu_stage`, so these changes are just replacing the `'t'` with a `'g'`.